libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -164,34 +145,93 @@ interface IIdent {
 }
 
 function extractCci(input: IIdent | IIdent[]): string[] {
-  let inputArray;
-  if (Array.isArray(input)) {
-    inputArray = input;
-  } else {
-    inputArray = [input];
-  }
+  const inputArray = asArray(input);
 
   const CCI_REGEX = /CCI-(\d*)/;
 
   const output: string[] = [];
-  inputArray.forEach((element) => {
+  for (const element of inputArray) {
     const text = _.get(element, 'text');
-    if (text.match(CCI_REGEX)) {
+    if (!!text && CCI_REGEX.exec(text)) {
       output.push(text);
     }
-  });
+  }
   return output;
 }
 
 function nistTag(input: IIdent | IIdent[]): string[] {
```

Comment:
<img width="689" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/d291373d-391c-413a-9909-dbc0fcbefbb7">

I think the function in that utility file is expecting the nist tag to only be ex. 'SI-2' whereas we are passing in ex. 'SI-2 (b)'.  Considering that the "800-53 Controls & CCIs" column in the table can show the full thing, I think this is a bug with the utility file where it's not dealing with the full string properly.  If you have the time to debug it, I'd appreciate it, otherwise I'll get to it myself in the next couple days.

Reply:
Have you had a chance to address this issue?

Reply:
Turns out that it was not a problem with the specification but with the zero padding that was causing issues.  Anyways, I've made the fix.  I will merge if there are no other issues that arise during the CI.